### ISSUE TITLE
What is the default path for files saved to Drive?

### ISSUE THREAD

**Issue by DavidPeterWlms** [GoogleChromeLabs/text-app#467]
Suddenly, I can't find my files! Any help would be appreciated

**Comment by BugsNash** [comment]
Hi DavidPeterWlms

When you save files from the text app into drive you choose the path that you save them in, I guess the 'default' would be the root folder in Drive (if you just click on the Drive folder to save there). I am able to save files from Text to Drive and recover them from a Pixelbook. They are also synced with Drive online as expected.  

Are you saying that when you try to save files to Drive from the Text app they don't appear at all? Or they do but then disappear after some time? Are you able to save/move files to Drive from other apps? E.g. if you drag a file to Drive from the Files app what happens?

**Comment by DavidPeterWlms** [comment]
Hi Bugs. Thanks for the love. Turns out that I had created/forgotten a
secondary level folder in Drive named "Txt Files". So when I scanned the
list of first level folders, nothing came up. So I recalled that one of the
files had 'std' in the name, which a Drive search pulled up and lead to the
forgotten folder.
Now the mystery is why the connection to that folder got broken.
Regardless, I have found my files!
Thanks for wanting to help, friend.
Peace
